org.winehq.Wine.yml

```diff
@@ -115,6 +115,10 @@ separate-locales: false
 cleanup:
   - '*.a'
   - '*.la'
+  - /*/cmake
+  - /*/pkgconfig
+  - /include
```

Comment:
These should be kept in place for a baseapp, since inherited apps may reuse some of the wine dependencies.